### ISSUE TITLE
PodSecurityPolicy is removed from >= 1.25

### ISSUE THREAD

**Issue by mike-code** [metal-stack/helm-charts#97]
csi-driver-lvm chart is still using `PodSecurityPolicy` which is removed in current k8s version

https://kubernetes.io/docs/concepts/security/pod-security-policy/

**Comment by majst01** [comment]
This is correct, we sadly still have some customers with ancient kubernetes versions, but we are getting there.
